ios/engine/KMEI/KeymanEngine/Classes/InputViewController.swift

```diff
@@ -22,10 +22,10 @@ public enum MenuBehaviour {
 }
 
 open class InputViewController: UIInputViewController, KeymanWebDelegate {
-  var menuCloseButtonTitle: String?
-  var isInputClickSoundEnabled = true
-  var globeKeyTapBehaviour = GlobeKeyTapBehaviour.switchToNextKeyboard
-  var menuBehaviour = MenuBehaviour.showAlways
+  public var menuCloseButtonTitle: String?
+  public var isInputClickSoundEnabled = true
+  public var globeKeyTapBehaviour = GlobeKeyTapBehaviour.switchToNextKeyboard
+  public var menuBehaviour = MenuBehaviour.showAlways
```

Comment:
These could probably be `protected` rather than `public` but I don't think it really hurts either way